src/stories/Counter.stories.ts

```diff
@@ -0,0 +1,24 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Counter from '../components/counter/Counter';
```

Comment:
There is no such file in your peoject!:
![Screenshot 2024-03-10 210704](https://github.com/weam1995/ReactMentoringProject/assets/79011727/d75d4196-a6c0-4ece-9e0f-53c7d310ba38)